profiles/static/styles/login.css

```
@@ -0,0 +1,63 @@
+    body,
+    html {
+        margin: 0;
+        padding: 0;
+        height: 100%;
+        background: #7abecc !important;
+    }
+    .user_card {
```

A general convention is class names have hyphens(`-`) in their names and ids have underscores(`_`) in their names. Change all class names and ids accordingly.
```suggestion
    .user-card {
```